### PR TITLE
samples: fix samples build

### PR DESCRIPTION
**Pull request by chingor13**
Snippets samples artifact will use the released artifact directly until
this library is available in libraries-bom (post-GA). Updating the
README accordingly.

### CODE REVIEW

**Review by stephaniewang526 — APPROVED** [review]

File: synth.py [hunk 1]
```diff
@@ -31,4 +31,6 @@
       destination_name='os-config',
   )
 
-java.common_templates()
\ No newline at end of file
+java.common_templates(excludes=[
```
Comment (kolea2) [inline synth.py:34]: do you need to exclude the readme too?

Comment (chingor13) [reply synth.py:34]: This one is an actual client where we do want to automatically generate the README.

Comment (chingor13) [reply synth.py:34]: After the client goes 1.0.0 and is included in the libraries-bom via google-cloud-bom, we will add back the sample and the README will show the libraries-bom install method.

Comment (kolea2) [reply synth.py:34]: won't this deleted section get regenerated? https://github.com/googleapis/java-os-config/pull/30/files#diff-04c6e90faac2675aa89e2176d2eec7d8L16 or am I missing something?

**Pull request merged**